scripts/main.py

```diff
@@ -285,8 +285,7 @@ def parse_arguments():
 if not cfg.pinecone_api_key or not cfg.pinecone_region: raise Exception("Please provide pinecone_api_key and pinecone_region")
```

Comment:
This needs to be updated. 
It doesn't make sense to force people to have a pinecone API key if pinecone is optional.

Reply:
Yeah it could probably be deleted but it doesn't seem to actually be blocking me.

Reply:
I went ahead and deleted it.